app/validate_txs.go

```diff
@@ -31,46 +63,40 @@ func separateTxs(_ client.TxConfig, rawTxs [][]byte) ([][]byte, []*tx.BlobTx) {
 	return normalTxs, blobTxs
 }
 
-// FilterTxs applies the antehandler to all proposed transactions and removes
-// transactions that return an error.
-//
-// Side-effect: arranges all normal transactions before all blob transactions.
-func FilterTxs(logger log.Logger, ctx sdk.Context, handler sdk.AnteHandler, txConfig client.TxConfig, txs [][]byte) [][]byte {
-	normalTxs, blobTxs := separateTxs(txConfig, txs)
-	normalTxs, ctx = filterStdTxs(logger, txConfig.TxDecoder(), ctx, handler, normalTxs)
-	blobTxs, _ = filterBlobTxs(logger, txConfig.TxDecoder(), ctx, handler, blobTxs)
-	return append(normalTxs, encodeBlobTxs(blobTxs)...)
-}
+func (fb *FilteredBuilder) Fill(txs [][]byte) [][]byte {
```

Comment:
the logic here is almost identical to the two functions we had before

however now it includes adding and reverting txs to the square.Builder